corehq/apps/domain/management/commands/make_superuser.py

```diff
@@ -28,11 +34,14 @@ def get_password_from_user():
                 return password
 
     @signalcommand
-    def handle(self, username, **options):
+    def handle(self, username, allow_superuser_management, **options):
         if not settings.ALLOW_MAKE_SUPERUSER_COMMAND:
             from dimagi.utils.web import get_site_domain
             raise CommandError(f"""You cannot run this command in SaaS Enviornments.
             Use https://{get_site_domain()}/hq/admin/superuser_management/ for granting superuser permissions""")
+        if not settings.ALLOW_SUPERUSER_MANAGEMENT:
+            raise CommandError('Assigning can_assign_superuser privilege has been disabled')
+            # reword this
```

Comment:
I'm not sure we need this separate setting (`ALLOW_SUPERUSER_MANAGEMENT`). I think the behavior we want is if `ALLOW_MAKE_SUPERUSER_COMMAND` is set to `True` that allows this management command to be run, period (with or without the `--allow-superuser-management` flag. I don't think there's much benefit to having it this granularly configurable.